### PR TITLE
feat(sdk-coin-bera): add TSS Suport

### PR DESCRIPTION
**Pull request by adarsh312**
Ticket: WIN-4073

### CODE REVIEW

**Review by dpkjnr — COMMENTED** [review]
there should be some tests around DKLS and walletVersion 5, can you add bera also there. This coinfeature also needs to be added `CoinFeature.MPCV2`

**Comment by dpkjnr** [comment]
> there should be some tests around DKLS and walletVersion 5, can you add bera also there. This coinfeature also needs to be added `CoinFeature.MPCV2`

this is one test https://github.com/BitGo/BitGoJS/blob/5360e06bca19430324e48e30ce5d0e4df1d79125/modules/bitgo/test/v2/unit/wallets.ts#L904

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -386,7 +386,8 @@ const ZKETH_FEATURES = [
 ];
 const BERA_FEATURES = [
   ...ETH_FEATURES,
-  CoinFeature.MULTISIG_COLD,
+  CoinFeature.TSS,
```
Comment (ajays97) [inline modules/statics/src/coins.ts:389]: Does this mean that bera started out as multisig and we are switching to TSS before rolling it out?

Comment (adarsh312) [reply modules/statics/src/coins.ts:389]: yes, we are switching to TSS. this decision is made before moving to mainnet

**Review by dpkjnr — APPROVED** [review]

**Pull request merged**